omnibus/package-scripts/iot-agent/postinst

```diff
@@ -54,6 +54,26 @@ if [ "$DISTRIBUTION_FAMILY" = "Debian" ]; then
 	esac
 	#DEBHELPER#
 
+	# Set the installation information if not already present;
+	if [ ! -f "$CONFIG_DIR/install_info" ]; then
+
+		if command -v dpkg >/dev/null 2>&1; then
+			tool=dpkg
+			tool_version=dpkg-$(dpkg --version | head -n 1 | cut -d " " -f 7 || echo "unknown")
+		else
+			tool=unknown
+			tool_version=unknown
+		fi
+		
+		install_info_content="---
+install_method:
+  tool: $tool
+  tool_version: $tool_version
+  installer_version: deb_package-iot
```

Comment:
There should also be an rpm_package-iot somewhere.

Reply:
Right now the rpm iot doesn't work (it will after #5444 gets merged). I think it is best if we wait until it gets merged and then I will add it.